dds/DCPS/transport/framework/TransportSendStrategy.cpp

```diff
@@ -935,7 +935,7 @@ TransportSendStrategy::send(TransportQueueElement* element, bool relink)
             "this->max_size_ == [%d].\n",
             this->max_size_));
 
-      const size_t max_message_size = this->max_message_size(element);
+      const size_t max_message_size = this->max_message_size();
```

Comment:
This was part of a plan to make the size adjustments conditional, but I'm fine with taking it out.